ios/Classes/PTFlutterViewController.m

```diff
@@ -150,4 +154,60 @@ - (void)pdfViewCtrl:(PTPDFViewCtrl *)pdfViewCtrl downloadEventType:(PTDownloaded
     [super pdfViewCtrl:pdfViewCtrl downloadEventType:type pageNumber:pageNum message:message];
 }
 
+#pragma mark - Viewer Settings
+
+- (void)initViewerSettings
+{
+    [self setReadOnly:NO];
+    [self setThumbnailViewEditingEnabled:YES];
+}
+
+- (void)applyViewerSettings
+{
+    [self applyReadOnly];
+    
+    // Thumbnail editing enabled.
+    self.thumbnailsViewController.editingEnabled = self.thumbnailViewEditingEnabled;
```

Comment:
Is there always a 1-to-1 mapping between the Flutter property `thumbnailViewEditingEnabled`  and the SDK's property thumbnailsViewController.editingEnabled?

If so, you can just make the Flutter one a facade for the SDK's:
```
-(BOOL)isThumbnailViewEditingEnabled
{
  return self.thumbnailsViewController.editingEnabled;
}

-(void)setThumbnailViewEditingEnabled:(BOOL)enabled
{
  return self.thumbnailsViewController.editingEnabled = enabled;
}
```

This would apply to any other property that behaves in this was (`annotationAuthor`?).